protobufs/livekit_room.proto

```diff
@@ -61,10 +61,9 @@ service RoomService {
   // Update room metadata, will cause updates to be broadcasted to everyone in the room, Requires `roomAdmin`
   rpc UpdateRoomMetadata (UpdateRoomMetadataRequest) returns (Room);
 
-  // Forward a participant's track to another room. Requires `roomAdmin` on the room whose name 
-  // consists of the source room and destination room as `source|dest`. The forwarding will stop 
-  // when the participant leaves the room or call `RemoveParticipant` on the destination room.
-  // A participant can be forwarded to multiple rooms. The destination room will be created if it does not exist.
+  // Forward a participant's track to another room. Requires `roomAdmin`. The forwarding will stop when the participant leaves the room
+  // or call this method again with `stop` set to true. A participant can be forwarded to multiple rooms. The destination room will be
```

Comment:
There is no more `stop` right? I thought we decided that `RemoveParticipant` should be used to stop forwarding.